data/resources/style.css

```diff
@@ -148,6 +148,48 @@ sidebarsearchitemrow {
   color: @accent_fg_color;
 }
 
+messagedocument > label.sender-text {
+  margin-bottom: 3px;
+}
+
+messagedocument > box {
+  padding: 9px;
+  transition: 200ms;
+}
+
+messagedocument > box > button {
+  background: @accent_color;
+  color: @window_bg_color;
+  min-width: 40px;
+  min-height: 40px;
+}
+
+messagedocument.outgoing > box > button {
+  background: @accent_fg_color;
+  color: @accent_bg_color;
+}
+
+messagedocument > box:not(.activatable) > button:hover {
+  background: mix(@accent_color, currentColor, 0.2);
+}
+
+messagedocument.outgoing > box:not(.activatable) > button:hover {
+  background: mix(@accent_fg_color, grey, 0.2);
+}
+
+messagedocument > box.activatable:hover {
+  background: alpha(currentColor, 0.1);
+}
+
```

Comment:
One more thing. This gives better user feedback that the message is actually clicked. It's similar when using `card` + `activatable` class. Not sure though whether `0.15` is the ideal value.
```suggestion

messagedocument > box.activatable:active {
  background: alpha(currentColor, 0.15);
}

```